osu.Game/Screens/Ranking/Expanded/Accuracy/AccuracyCircle.cs

```diff
@@ -195,14 +196,14 @@ private void load()
                             Blending = new BlendingParameters
                             {
                                 Source = BlendingType.DstColor,
-                                Destination = BlendingType.OneMinusSrcAlpha,
+                                Destination = BlendingType.OneMinusSrcColor,
```

Comment:
Please pay attention to this change, I'm not entirely sure if it's correct, just plugged-in "whatever works" because `OneMinusSrcAlpha` wasn't working as expected